server/@types/prisonRegisterImport/index.d.ts

```diff
@@ -14,14 +14,31 @@ export interface paths {
     put: operations['putEmailAddressForOffenderManagementUnit']
     delete: operations['deleteEmailAddressForOffenderManagementUnit']
   }
+  '/queue-admin/retry-dlq/{dlqName}': {
+    put: operations['retryDlq']
+  }
+  '/queue-admin/retry-all-dlqs': {
```

Comment:
obv all these changes are created from the npx openapi-typescript command.  Do we normally edit this file to remove things that aren't required or just leave it as it is?

Reply:
I was thinking this when I did it and figured it made sense to leave it as it is because otherwise everytime someone regenerates they'll need to manually prune this